src/integration-test/java/com/commercetools/sync/integration/inventories/InventorySyncIT.java

```diff
@@ -135,37 +136,6 @@ void sync_WithNewInventory_ShouldCreateInventory() {
         assertValues(oldInventoryAfterSync.get(), QUANTITY_ON_STOCK_2, EXPECTED_DELIVERY_2, RESTOCKABLE_IN_DAYS_2);
     }
 
-    @Test
```

Comment:
why did you need to delete this test method?

Reply:
because I noticed its a redundant test, it's testing nothing special compared to `sync_WithKeyToExistingSupplyChannelInPlaceOfReferenceId_ShouldUpdateInventory` for example.